lib/checkers.cpp

```diff
@@ -23,6 +23,8 @@
 
 namespace checkers {
     const std::map<std::string, std::string> allCheckers{
+        {"Check64BitPortability::pointerassignment","portability"},
```

Comment:
I want that these lines are sorted somehow so we will not get such diffs in the future but it doesn't have to be done in this PR.